server/src/main/java/org/opensearch/cluster/coordination/ClusterBootstrapService.java

```diff
@@ -128,11 +137,11 @@ public ClusterBootstrapService(
             bootstrapRequirements = Collections.singleton(Node.NODE_NAME_SETTING.get(settings));
             unconfiguredBootstrapTimeout = null;
         } else {
-            final List<String> initialMasterNodes = INITIAL_MASTER_NODES_SETTING.get(settings);
+            final List<String> initialMasterNodes = INITIAL_CLUSTER_MANAGER_NODES_SETTING.get(settings);
```

Comment:
Change the variable name? It still has the _master_ verbiage

Reply:
Really good catch!👍 I even missed the term `initialMasterNodes` in PR https://github.com/opensearch-project/OpenSearch/pull/2150, which aims to address these. 😂
I updated the name in a commit there https://github.com/opensearch-project/OpenSearch/pull/2150/commits/91d768cc49f4c650c845ac6cb57f5fce870390ba.